content_api/items/service.py

```diff
@@ -96,7 +96,9 @@ def get(self, req, lookup):
         self._set_fields_filter(req)  # Eve's "projection"
 
         try:
-            return super().get(req, lookup)
+            res = super().get(req, lookup)
+            req.args = request_params
```

Comment:
It might make sense to rename `request_params` to `orig_request_params` or something, so that it's more obvious that we are _restoring_ the value of `req.args` here.